app/src/main/java/com/everest/moviedb/ui/HomeFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.everest.moviedb.ui
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.*
+import androidx.fragment.app.Fragment
+import com.everest.moviedb.R
+import com.everest.moviedb.adapters.PageAdapter
+import com.everest.moviedb.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
```

Comment:
we can use `late init var`. so that we do not need to make it nullable

Reply:
if we make it nullable. we should not use force non-null conversion ( it will crash if app not able to convert this properly)
we should use like this : `_binding?.root`